### DISCUSSION TITLE
Server Error 500 when trying to remove the account and change subscription plan to free not possible!

### DISCUSSION THREAD

**Discussion by Rapsodia86** [opengisch/QField#3957] [Bugs]
Hello,
I signed up for QField almost two months ago, but after some time I decided to use other software. However, for the last couple of weeks, I was not able to remove my account (server error 500), now I ended up with a paid subscription (shows "pro" next to username) which I cannot change because I am getting another error: “Unable to process the request at the moment, remote system unavailable. Please try again later.”

username: mtomaszewska

May I ask for support?
Thanks

**Comment by suricactus** [comment]
Your account is now a community one.

Certain accounts are temporary unable to be deleted. Will be available again by later this week. Apologies for the inconvenience.

**Reply by Rapsodia86** [reply]
EDIT:
Ok, sorry! I just noticed that indeed the account was changed to the community one and is free! I appreciate that fast help! 

Thanks for the quick answer!
How about the paid subscription? I cannot change to a free one. That will also be possible later this week? I just do not want to be charged for a thing I do not use;)

Best,
M

**Reply by suricactus** [reply]
If you don't put your card, you cannot be charged in any way, no worries. ;) 

> How about the paid subscription? I cannot change to a free one. That will also be possible later this week? I just do not want to be charged for a thing I do not use;)

Not sure if I follow. Now you should have a community account. The pro you were seeing was by a mistake, being one of these "certain accounts".

Sorry for the inconvenience and the stress you are going to get charged for something you do not use.

**Reply by Rapsodia86** [reply]
Ok, everything is fine. I sent this "Thanks for the quick answer!
How about the paid subscription? I cannot change to a free one. That will also be possible later this week? I just do not want to be charged for a thing I do not use;)" and later I realized that the account has already been changed. That is why the EDIT.
As for being charged, I meant getting an invoice since my card was not provided;)

Thanks again for your swift action and all the explanation!
